vllm/entrypoints/openai/cli_args.py

```diff
@@ -306,7 +306,7 @@ def log_non_default_args(args: argparse.Namespace):
     for arg, default in vars(parser.parse_args([])).items():
         if default != getattr(args, arg):
             non_default_args[arg] = getattr(args, arg)
-    logger.info("non-default args: %s", non_default_args)
+    logger.debug("non-default args: %s", non_default_args)
```

Comment:
I like this one in info. It's a lot better than it used to be when it logged ALL args. This is very useful when someone reports an issue.